database/postgres/dbconnect.js

```diff
@@ -1,22 +1,10 @@
-const { Pool, Client } = require('pg');
+const { Pool } = require('pg');
 const pool = new Pool({
 	user: 'postgres',
 	database: 'sdcimages',
 	password: 'password',
 });
```

Comment:
should username/ password info eventually go on a file that is not committed up to github? 
Think it's fine for local development but I'd think about it in the context of production

Reply:
oh smart good point, yes it should! i'll make that change for the next PR